stubs/php-file-get-contents/README.md

```diff
@@ -0,0 +1,39 @@
+# How to run
+
+```console
+./run.php expired.badssl.com 443
+```
+
+# Dependencies
+
+ * php
```

Comment:
Also (I didn't test this but) [`allow_url_fopen`](https://secure.php.net/manual/en/filesystem.configuration.php) needs to be set `True` in php.ini.